spec/support/factories.rb

```diff
@@ -6,4 +6,24 @@
     password 'password'
     password_confirmation 'password'
   end
+
+  factory :art do
+    sequence(:id) { |n| }
+    name "Chinatown Community Mural"
+    location "Surface Rd & Beach St"
+    description "Chinese community mural commissioned
+    by the mayor's mural crew."
+    artist "Unknown"
+    category "mural"
+  end
+
+  factory :review do
+    sequence(:id) { |n| }
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.